code/game/supplyshuttle.dm

```diff
@@ -168,21 +168,14 @@ var/list/mechtoys = list(
 /datum/controller/supply/New()
 	ordernum = rand(1,9000)
 
-//Supply shuttle ticker - handles supply point regeneration and shuttle travelling between centcomm and the station
-/datum/controller/supply/proc/process()
 	for(var/typepath in subtypesof(/datum/supply_packs))
 		var/datum/supply_packs/P = new typepath()
-		if(P.name == "HEADER")
-			qdel(P)
-			continue
+		if(P.name == "HEADER") continue

```

Comment:
In this case, I actually need them created; the headers are what determines the access requirements for all their children crates, so they do need to be created, but not accessible to player.

Thanks for the suggestion though, it's much appreciated!